examples/application-composer/README_pt.adoc

```diff
@@ -0,0 +1,164 @@
+= Compositor de Aplicações
+:index-group: Testing Techniques
+:jbake-type: page
+:jbake-status: published
+
+O `org.apache.openejb.junit.ApplicationComposer` é o corredor de testes JUnit modelado de acordo com a forma como fizemos testes internos no OpenEJB por anos (desde 2006).
+Isto não envolve o escaneamento de toda a classpath.  Se você quiser alguma coisa na aplicação, você deve construí-lo diretamente no seu testcase.
+
+Com o `ApplicationComposer` você pode fazer testes idênticos ao OpenEJB que usa internamente, mas com dependência limitada do próprio OpenEJB.  A principal dependência está no código
```

Comment:
In this sentence: "você pode fazer testes idênticos ao OpenEJB que usa internamente" i believe that "você pode fazer testes idênticos ao que o OpenEJB  usa internamente"